bitnami/redis/templates/scripts-configmap.yaml

```
@@ -596,6 +603,16 @@ data:
         exit 0
     fi
 
+  {{- if .Values.sentinel.service.createMaster}}
+  push-master-label.sh: |
+    #!/bin/bash
+    # https://download.redis.io/redis-stable/sentinel.conf
+
+    PREVIOUS="${4}"
+    CURRENT="${6}"
+    echo "kubectl label pod --field-selector metadata.name=${CURRENT/.*} isMaster=\"true\" --overwrite
+    kubectl label pod --field-selector metadata.name=${PREVIOUS/.*} isMaster=\"false\" --overwrite" > /kubectl/update-annotations.sh
```

Can we simply share "PREVIOUS" and "CURRENT" values with the sidecar container?
```suggestion
    #!/bin/bash
    # https://download.redis.io/redis-stable/sentinel.conf

    echo "${4/.*}" > /kubectl/previous
    echo "${6/.*}" > /kubectl/current
```